### PR TITLE
Add support for new languages in properties object

### PR DESCRIPTION
**Pull request by mattieruth**

### CODE REVIEW

File: src/module.js [hunk 1]
```diff
@@ -118,8 +118,10 @@ const FRAME_PROPS = {
   },
   lang: { 
     validate: (lang) => {
-      return lang === 'en-us' || lang === 'en' || lang === 'fr';
-    }
+      return lang === 'de' || lang === 'en-us' || lang === 'en' || lang === 'fi' ||
```
Comment (kompfner) [inline src/module.js:121]: might be easier to maintain following the pattern `return ['de','en-us','en','fi'].includes(lang);`

File: src/module.js [hunk 2]
```diff
@@ -118,8 +118,10 @@ const FRAME_PROPS = {
...
+        lang === 'fr' || lang === 'nl' || lang === 'pt' || lang === 'user';
+    },
+    help: 'language not supported. Options are: de, en-us, en, fi, fr, nl, pt, user'
```
Comment (kompfner) [inline src/module.js:124]: 👍

File: src/module.js (hunk 1 above)
Comment (mattieruth) [reply src/module.js:121]: oh that's nice. i'll do that. i forget about `includes`

Comment (mattieruth) [reply src/module.js]: updated to your suggestion and in the process found out that 'user' is only an option at the domain/room rest api level. Not from the front-end... so I've removed that.

**Pull request merged**